daemon/api.go

```diff
@@ -2146,15 +2158,15 @@ func runSnapctl(c *Command, r *http.Request, user *auth.UserState) Response {
 		return BadRequest("snapctl cannot run without args")
 	}
 
-	_, uid, _, err := runSnapctlUcrednetGet(r.RemoteAddr)
+	ucred, err := runSnapctlUcrednetGet(r.RemoteAddr)
 	if err != nil {
 		return Forbidden("cannot get remote user: %s", err)
```

Comment:
Nitpick, perhaps this should say `requesting user` instead of `remote` as we may eventually have authenticated remote access over non-local sockets.